pyproject.toml

```diff
@@ -35,10 +36,6 @@ test = [
 pythonpath = "src"
 
 [tool.pytest.ini_options]
-log_cli = true
```

Comment:
I didn't know we had this, but am wondering if this change is intentional or not?

Reply:
I think @ehsavoie added this in another PR